packages/ses/src/error/tame-console.js

```diff
@@ -17,11 +18,13 @@ const originalConsole = console;
  *
  * @param {"safe" | "unsafe"} consoleTaming
  * @param {"platform" | "exit" | "abort" | "report" | "none"} [errorTrapping]
+ * @param {"platform" | "exit" | "abort" | "report" | "none"} [unhandledRejectionTrapping]
  * @param {GetStackString=} optGetStackString
  */
 export const tameConsole = (
   consoleTaming = 'safe',
   errorTrapping = 'platform',
+  unhandledRejectionTrapping = 'platform',
```

Comment:
As mentioned elsewhere, do we want `'platform'` to be the default? In any case, glad that `'none'` isn't the default after all. That was scary ;)

Also mentioned elsewhere: An option named `'platform'` should provide the actual platform behavior.